kuma/static/img/subscriptions/money.svg

```diff
@@ -0,0 +1,3 @@
+<svg class="icon icon-money-bill" xmlns="http://www.w3.org/2000/svg" width="23" height="16" aria-hidden="true">
```

Comment:
We should remove the `width` and `height` and replace with `viewBox` for responsiveness(especially in IE11) - Also, here and the one above, we should add `role="img"`